src/ansys/fluent/core/session_solver.py

```diff
@@ -271,13 +271,15 @@ def _populate_settings_api_root(self):
 
     def __getattr__(self, attr):
         self._populate_settings_api_root()
+        if attr in [x for x in dir(self._settings_api_root) if not x.startswith("_")]:
+            warnings.warn(
+                f"'{attr}' is deprecated. Use 'settings.{attr}' instead.",
+                DeprecationWarning,
```

Comment:
Please use a custom class for this warning, so that it can be easily disabled if ever needed.

Reply:
There are currently custom warnings `DeprecatedSettingWarning` and `UnstableSettingWarning` for settings API. The formaer can be used for the currrent purpose.